commit 47127f34e7478ef369b2f14d820a375e368deb5b
Author: Henry Ptasinski <henryp@broadcom.com>
Date:   Fri Oct 1 14:05:20 2010 -0700

    staging: brcm80211: Remove unused structure.
    
    Remove the wl_assoc_info_t structure.  It's never used, and depends on
    structures defined in other header files that can now also be cleaned up.
    
    Signed-off-by: Henry Ptasinski <henryp@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/wlioctl.h
+++ b/drivers/staging/brcm80211/include/wlioctl.h
@@ -622,15 +622,6 @@ typedef struct wl_led_info {
 	uint8 activehi;
 } wl_led_info_t;
 
-typedef struct wl_assoc_info {
-	uint32 req_len;
-	uint32 resp_len;
-	uint32 flags;
-	struct dot11_assoc_req req;
-	struct ether_addr reassoc_bssid;	/* used in reassoc's */
-	struct dot11_assoc_resp resp;
-} wl_assoc_info_t;
-
 /* flags */
 #define WLC_ASSOC_REQ_IS_REASSOC 0x01	/* assoc req was actually a reassoc */
 
